### ISSUE TITLE
Have facts return the name of the node as a separate line item

### ISSUE THREAD

**Issue by mdobeck1** [F5Networks/f5-ansible#393]
##### ISSUE TYPE

 - Feature Idea

##### COMPONENT NAME

bigip_facts.py
##### ANSIBLE VERSION

```

```

##### PYTHON VERSION

```

```

##### BIGIP VERSION

```

```

##### SUMMARY

Have _bigip_facts.py return the name of the node as a separate line in json rather than the top level heading. it would make it infinitely easier to reference.
##### STEPS TO REPRODUCE

```node:
  /Common/node01:
    address: "1.2.3.6"
    connection_limit: 0
    description: ""
    dynamic_ratio: 1
    monitor_instance: ~
    monitor_rule:
      monitor_templates: ~
      quorum: 0
      type: MONITOR_RULE_TYPE_NONE
    monitor_status: MONITOR_STATUS_UNCHECKED
    object_status:
      availability_status: AVAILABILITY_STATUS_BLUE
      enabled_status: ENABLED_STATUS_ENABLED
      status_description: "Node address does not have service checking enabled"
    rate_limit: 0
    ratio: 1
    session_status: SESSION_STATUS_ENABLED
```
vs
```node:
  /Common/node01:
    name: "node01"
    address: "1.2.3.6"
    connection_limit: 0
    description: ""
    dynamic_ratio: 1
    monitor_instance: ~
    monitor_rule:
      monitor_templates: ~
      quorum: 0
      type: MONITOR_RULE_TYPE_NONE
    monitor_status: MONITOR_STATUS_UNCHECKED
    object_status:
      availability_status: AVAILABILITY_STATUS_BLUE
      enabled_status: ENABLED_STATUS_ENABLED
      status_description: "Node address does not have service checking enabled"
    rate_limit: 0
    ratio: 1
    session_status: SESSION_STATUS_ENABLED
  /Common/node01:```
```

##### ACTUAL RESULTS

```

```

**Comment by caphrim007** [comment]
PR created for this upstream https://github.com/ansible/ansible/pull/32165

**Comment by caphrim007** [comment]
@mdobeck1 this is now merged and will be released with ansible 2.5
